golem/resources/base.py

```diff
@@ -52,14 +53,14 @@ class _NULL:
 
 
 def api_call_wrapper(
-    ignore_status_codes: List[int] = [],
+    ignore_status_codes: Tuple[int, ...] = (),
```

Comment:
actually, the hint type _could_ be anything iterable...

Reply:
...

Reply:
?

Reply:
as in:

```suggestion
    ignore_status_codes: typing.Iterable[int, ...] = (),
```